mockery/generator_test.go

```diff
@@ -302,6 +302,18 @@ type RequesterPtr struct {
 	mock.Mock
 }
 
+type RequesterPtr_Get struct {
+	*mock.Call
+}
+
+func (_m *RequesterPtr_Get) Return (str *string, err error) {
+}
+
+func (_m *RequesterPtr) OnGet(path string) *RequesterPtr_Get {
+	c := _m.On("Get")
```

Comment:
Not sure I understand this. Don't you want this?

```go
func (_m *RequesterPtr) OnGet(path string) *RequesterPtr_Get {
	return _m.On("Get", path)
}
```

I don't understand the need for the `RequesterPtr_Get` struct.